### PR TITLE
HMIS-1209 Spike for terraform

### PR DESCRIPTION
**Pull request by junaidiqbalmoj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/HMIS-1209

### Change description ###

Add multiple policy files for endpoints.

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[x] No

### CODE REVIEW

File: .idea/.gitignore
Comment (joshblackmoor) [inline .idea/.gitignore:1]: .idea/ folder should be included in the gitignore

**Review by joshblackmoor — COMMENTED** [review]
Policy files will need updating but assuming that will happen in the later tickets

**Review by joshblackmoor — COMMENTED** [review]
You also need to do the infra approvals before merge. See message in pipeline regarding it

File: .idea/.gitignore
Comment (junaidiqbalmoj) [reply .idea/.gitignore:1]: added in gitignore.

**Review by junaidiqbalmoj — COMMENTED** [review]
sds infra changes have been approved and pipeline is green now.

**Review by joshblackmoor — COMMENTED** [review]
Code looks fine, can still see .idea folder though

File: resources/api-spec/hmi-api-health.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "hmi-apim-health-api",
+    "version": "1.0"
+  },
+  "host": "hmi-apim.dev.platform.hmcts.net",
```
Comment (ChrisS1512) [inline resources/api-spec/hmi-api-health.json:7]: Should this host be configurable / it replaces the dev with the relevant env it's running in?

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: resources/api-spec/hmi-api-health.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "description": "Liveness Check",
+        "operationId": "liveness-check",
+        "summary": "Liveness Check",
+        "parameters": [],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "",
```
Comment (ChrisS1512) [inline resources/api-spec/hmi-api-health.json:22]: Can a description be added for this

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: resources/policy-files/CaTH/publication.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<policies>
+    <inbound>
+        <rewrite-uri template="/" copy-unmatched-params="true" />
+        <set-backend-service base-url="https://hmi-apim.test.publication.platform.hmcts.net/hmi/" />
```
Comment (ChrisS1512) [inline resources/policy-files/CaTH/publication.xml:4]: Similar here (and the api-policy.xml file)

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

**Review by joshblackmoor — APPROVED** [review]

**Pull request merged**